modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/test/java/com/liferay/portal/workflow/metrics/internal/scheduler/WorkflowMetricsSLADefinitionTransformerSchedulerJobConfigurationTest.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.workflow.metrics.internal.scheduler;
+
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.PropsKeys;
+import com.liferay.portal.kernel.util.PropsUtil;
+import com.liferay.portal.search.capabilities.SearchCapabilities;
+import com.liferay.portal.search.engine.adapter.SearchEngineAdapter;
+import com.liferay.portal.search.engine.adapter.index.IndicesExistsIndexRequest;
+import com.liferay.portal.search.engine.adapter.index.IndicesExistsIndexResponse;
+import com.liferay.portal.search.engine.adapter.search.SearchSearchRequest;
+import com.liferay.portal.search.engine.adapter.search.SearchSearchResponse;
+import com.liferay.portal.search.hits.SearchHits;
+import com.liferay.portal.search.index.IndexNameBuilder;
+import com.liferay.portal.search.query.BooleanQuery;
+import com.liferay.portal.search.query.Queries;
+
+import java.lang.reflect.Method;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * @author Aquiles Duarte
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class
+	WorkflowMetricsSLADefinitionTransformerSchedulerJobConfigurationTest {
+
+	@Test
+	public void testTransformReadsIndexSearchLimitAndCallsTransformer()
```

Comment:
Where did you find this naming pattern? 🤔
I think a better name for this test would simply be testTransform().